src/lib/jsonParse/jsonRequest.cpp

```diff
@@ -199,20 +198,14 @@ std::string jsonTreat
 
   if (reqP == NULL)
   {
-    std::string errorReply =
-      restErrorReplyGet(ciP,
-                        "",
-                        requestType(request),
-                        SccBadRequest,
-                        std::string("Sorry, no request treating object found for RequestType /") +
-                        requestType(request) + "/");
-
-
-    char reqTypeV[STRING_SIZE_FOR_INT];
+    std::string  details = std::string("Sorry, no request treating object found for RequestType /") + requestType(request) + "/";
+    std::string  errorReply;
+    char         reqTypeV[STRING_SIZE_FOR_INT];
 
+    restErrorReplyGet(ciP, SccBadRequest, details, &errorReply);
     snprintf(reqTypeV, sizeof(reqTypeV), "%d", request);
-    std::string details = std::string("no request treating object found for RequestType ") + reqTypeV + " (" + requestType(request) + ")";
-    alarmMgr.badInput(clientIp, details);
+    std::string details2 = std::string("no request treating object found for RequestType ") + reqTypeV + " (" + requestType(request) + ")";
```

Comment:
Why don't reuse `details` variable as in the former implementation?

Not very important... but I'm curiosious about the change :)

Reply:
No reason. I will check it

Reply:
Guess I got a compilation error that `details` was already defined and I just changed the name

Reply:
Fixed in 1e2e62c